docs/.vitepress/vite-plugin.mts

```diff
@@ -58,19 +59,22 @@ function transformRequire(code: string) {
       }
 
       let id =
+        // eslint-disable-next-line prefer-template
         '__' +
         moduleString.replace(/[^a-zA-Z0-9_$]+/gu, '_') +
-        Math.random().toString(32).substring(2)
+        Math.random().toString(32).slice(2)
       while (code.includes(id) || modules.has(id)) {
-        id += Math.random().toString(32).substring(2)
+        id += Math.random().toString(32).slice(2)
       }
       modules.set(id, moduleString)
-      return id + '()'
+      return `${id}()`
     }
   )
 
   return (
+    // eslint-disable-next-line prefer-template
     [...modules]
+      // eslint-disable-next-line arrow-body-style
```

Comment:
Please remove this inline disable comment regarding `arrow-body-style` and instead fix the issue. [My comment above](https://github.com/vuejs/eslint-plugin-vue/pull/2960#discussion_r2546288430) was referring to the `prefer-template` rule only.